install-pre-commit-hook.sh

```diff
@@ -1,46 +0,0 @@
-#!/bin/bash
```

Comment:
We now recommend using https://github.com/BrainMaestro/composer-git-hooks with Composer or https://github.com/typicode/husky with npm.

Reply:
Does it hurt to continue bundling this? I would think it would be better to just add a deprecation notice to `echo` at the top of the script and advise using the Composer/NPM alternatives.

Reply:
You're right -- we can leave it in because it uses only relative path in the logic!